core_utils/article/article.py

```diff
@@ -180,7 +180,7 @@ def get_cleaned_text(self) -> str:
         Returns:
             str: Cleaned text.
         """
-        return self.text.lower().translate(str.maketrans('', '', string.punctuation))
+        return self.text.lower().translate(str.maketrans('', '', string.punctuation + '()[]{}«»'))
```

Comment:
i'll remove it because it does not influence on the tests. i wanted include this because in the udpipe files tokens is comprised of some extra punctuation. however, it does not influence on parsing. at the same time, theare are no such defects in stanza files because it has more functionalities.